.vscode/recommended.settings.json

```diff
@@ -4,5 +4,8 @@
     "editor.formatOnSave": true,
     "python.testing.pytestEnabled": true,
     "python.linting.flake8Enabled": true,
-    "python.linting.mypyEnabled": true
+    "python.linting.mypyEnabled": true,
+    "python.testing.pytestArgs": [],
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false
```

Comment:
I should just copy these settings into my `settings.json` right?

Reply:
If you like!

If you do, you will get this to appear in VSCode:

![image](https://user-images.githubusercontent.com/34773432/108650681-f592d080-7474-11eb-8909-5b1c2e1bd8cd.png)